_health-care/_js/components/insurance-information/MedicareMedicaidSection.jsx

```diff
@@ -12,11 +15,10 @@ class MedicareMedicaidSection extends React.Component {
 
         <div className="row">
           <div className="small-12 columns">
-            <input
-                id="veteran_is_medicaid_eligible"
-                name="veteran_is_medicaid_eligible"
-                type="checkbox"/>
-            <label htmlFor="veteran_is_medicaid_eligible">Are you eligible for medicaid?</label>
+            <ErrorableCheckbox
+                label="Are you eligible for medicaid?"
```

Comment:
I know this is just copied from the original markup, but should "medicaid" be capitalized?